types/personalDataErasure.js

```diff
@@ -0,0 +1,199 @@
+/* @flow */
+
+/* Logger */
+export type LoggerOptions = {
+  error: Function,
```

Comment:
Nitpick: Instead of typing this as `Function`, it’s better to specify the full signature with the return value of `unit`.